prime-router/src/main/kotlin/azure/BatchFunction.kt

```diff
@@ -51,10 +51,14 @@ class BatchFunction {
                     actionHistory.trackExistingInputReport(it.task.reportId)
                     report
                 }
-                val outReports = when (receiver.batch?.operation) {
+                val mergedReports = when (receiver.batch?.operation) {
                     OrganizationService.BatchOperation.MERGE -> listOf(Report.merge(inReports))
                     else -> inReports
```

Comment:
See my next comment below.   I think Batch should just always merge, and nothing else.

If some receiver doesn't want merging, then simply skip that step in the workflow.

Reply:
Disagree: batch is what happens at a later time. It should do all the things that should happen at a later time.